app/src/main/java/org/sopt/at/MyActivity.kt

```diff
@@ -0,0 +1,36 @@
+package org.sopt.at
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import org.sopt.at.ui.theme.ATSOPTANDROIDTheme
+
+class MyActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val userId = intent.getStringExtra("userId") ?: ""
+        enableEdgeToEdge()
+        setContent {
+            ATSOPTANDROIDTheme {
+                Scaffold(modifier = Modifier.fillMaxSize()) { innerPadding ->
+                    MyView(Modifier.padding(innerPadding), userId)
```

Comment:
```suggestion
                    MyView(
                      modifier = Modifier.padding(innerPadding),
                      userId = userId
                      )
```
이런 식으로 어떤 인자에 어떤 값을 넣는지 명시해주는게 좋습니다!
이러면 코드의 가독성이 좋아지죠! 어떤 속성에 어떤 값을 넣어준건지 확인하기도 쉽고, 실수도 많이 줄어듭니다